### ISSUE TITLE
Migrations in cm are run twice

### ISSUE THREAD

**Issue by alexispeter** [cargomedia/cm#2566]
When running the migration cli command or executing tests in cm, all migration-scripts are handled twice.

The problem lies within CM_Migration_Manager::_getMigrationPaths().
```php
protected function _getMigrationPaths() {
    $paths = [
        self::getMigrationPathByModule(),
    ];
    foreach ($this->_modules as $moduleName) {
        $paths[] = self::getMigrationPathByModule($moduleName);
    }
    return $paths;
}
```
Both `self::getMigrationPathByModule()` and `self::getMigrationPathByModule($moduleName)` return `/home/vagrant/cm//resources/migration` which causes the directory to be scanned twice.

The exception being thrown
```
CM_Db_Exception: Cannot execute SQL statement in /home/vagrant/cm/library/CM/Db/Statement.php on line 47
     0. {main} /home/vagrant/cm/vendor/phpunit/phpunit/phpunit:0
     1. PHPUnit_TextUI_Command->main() /home/vagrant/cm/vendor/phpunit/phpunit/phpunit:47
     2. PHPUnit_TextUI_Command->run([], true) /home/vagrant/cm/vendor/phpunit/phpunit/src/TextUI/Command.php:115
     3. PHPUnit_TextUI_Command->handleArguments([]) /home/vagrant/cm/vendor/phpunit/phpunit/src/TextUI/Command.php:126
     4. PHPUnit_TextUI_Command->handleBootstrap('/home/vagrant/cm/tests/bootstr...') /home/vagrant/cm/vendor/phpunit/phpunit/src/TextUI/Command.php:731
     5. PHPUnit_Util_Fileloader->checkAndLoad('/home/vagrant/cm/tests/bootstr...') /home/vagrant/cm/vendor/phpunit/phpunit/src/TextUI/Command.php:902
     6. PHPUnit_Util_Fileloader->load('/home/vagrant/cm/tests/bootstr...') /home/vagrant/cm/vendor/phpunit/phpunit/src/Util/Fileloader.php:38
     7. include_once('/home/vagrant/cm/tests/bootstr...') /home/vagrant/cm/vendor/phpunit/phpunit/src/Util/Fileloader.php:56
     8. CMTest_TestSuite->bootstrap() /home/vagrant/cm/tests/bootstrap.php:14
     9. CMTest_TH->init() /home/vagrant/cm/tests/helpers/CMTest/library/CMTest/TestSuite.php:14
    10. CM_Provision_Loader->load(CM_OutputStream_Null) /home/vagrant/cm/tests/helpers/CMTest/library/CMTest/TH.php:14
    11. CM_Db_SetupScript->load(CM_OutputStream_Null) /home/vagrant/cm/library/CM/Provision/Loader.php:34
    12. CM_Db_SetupScript->_setInitialMigrationScripts() /home/vagrant/cm/library/CM/Db/SetupScript.php:18
    13. CM_Migration_Model->create('1486572082_ClockworkJobs') /home/vagrant/cm/library/CM/Db/SetupScript.php:71
    14. CM_Model_Abstract->commit() /home/vagrant/cm/library/CM/Migration/Model.php:58
    15. CM_Model_StorageAdapter_Database->create(48, []) /home/vagrant/cm/library/CM/Model/Abstract.php:89
    16. CM_Db_Db->insert('cm_migration', [], NULL, NULL, 'INSERT') /home/vagrant/cm/library/CM/Model/StorageAdapter/Database.php:49
    17. CM_Db_Query_Abstract->execute() /home/vagrant/cm/library/CM/Db/Db.php:133
    18. CM_Db_Statement->execute([]) /home/vagrant/cm/library/CM/Db/Query/Abstract.php:40
    19. {throw} /home/vagrant/cm/library/CM/Db/Statement.php:47
tries: 0
originalExceptionMessage: 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1486572082_ClockworkJobs' for key 'name''
query: 'INSERT INTO `cm_migration` (`name`) VALUES (?)'
```
fyi @zazabe

**Comment by alexispeter** [comment]
To reproduce, simply create a migration-script in cm and run any test.

**Comment by zazabe** [comment]
thanks for reporting this @alexispeter !

**Comment by zazabe** [comment]
fixed by #2568
